src/styles/theme/index.ts

```diff
@@ -137,12 +137,31 @@ export const common = {
         one: '#1EC490',
         two: '#FF961B',
         three: '#FC6A8A',
+        four: '#F17047',
       },
-      tokenomics: {
-        one: '#1EC490',
-        two: '#497BFF',
-        three: '#9F46EC',
-      },
+      tokenomics: [
```

Comment:
this is ok for static validator set, but any time new validator is added we would need to add hardcoded value and re-deploy explorer
it would be more helpful if we can randomly generate colors based on current data set, i did quick googling, not sure if this is a solution, but just as illustration:
```
for(let i=0;i<this.data.length;i++){
      this.colors.push('#'+Math.floor(Math.random()*16777215).toString(16));
}
```

Reply:
Yep, get it! I will try look it up